server/service/src/sync/translations/stocktake_line.rs

```diff
@@ -34,6 +34,7 @@ pub struct LegacyStocktakeLineRow {
     #[serde(deserialize_with = "empty_str_as_option_string")]
     pub item_line_ID: Option<String>,
     pub item_ID: String,
+    #[serde(default)]
     pub item_name: String,
```

Comment:
I get it for the options, but why would this need a default? Maybe we need to fix the reference data if we have stocktake lines that don't have items/item_names?

Reply:
Yeah, it's not ideal :) Especially for item_name maybe I take that one out but merge the other two?

Reply:
Remove this... I figure the rest is ok to have, and get's people up and running for now, until we can re-generate the data...